packages/react-wildcat-handoff/src/utils/defaultTemplate.js

```diff
@@ -35,13 +36,28 @@ module.exports = function defaultTemplate(cfg) {
 
         <link rel="prefetch" href="${staticUrl}/jspm_packages/system.js" />
         <link rel="prefetch" href="${staticUrl}/system.config.js" />
-        <link rel="prefetch" href="${staticUrl}/bundles/react.js" />
+        ${__PROD__ ? `<link rel="prefetch" href="${staticUrl}/bundles/react.js" />` : ``}
 
         ${helmetTags.join(``)}
     </head>
     <body>
         <div id="${reactRootElementID}">${html}</div>
-
+        ${serviceWorker ? `
+        <script src="/register-sw.js"></script>
+        ` : `
+        <script>
+            // Remove any registered service workers (dev mode only)
+            navigator.serviceWorker.getRegistrations().then(function(registrations) {
+                for(let registration of registrations) {
```

Comment:
Space after for. Use `var` not `let`, we need to dip down to ES5 for certain browsers.